patches/ui-webui-resources-tools-build_webui.gni.patch

```diff
@@ -1,20 +1,37 @@
 diff --git a/ui/webui/resources/tools/build_webui.gni b/ui/webui/resources/tools/build_webui.gni
-index 98fd56c9308df54eb0f3fa52997a80fd72efa503..c850c31ee115c007f1209abe694f7b62fe006a20 100644
+index 98fd56c9308df54eb0f3fa52997a80fd72efa503..6a4b42be7399d9326fe42aef6ae52c1b3645c1aa 100644
 --- a/ui/webui/resources/tools/build_webui.gni
 +++ b/ui/webui/resources/tools/build_webui.gni
-@@ -209,6 +209,7 @@ template("build_webui") {
+@@ -18,6 +18,7 @@ import("//ui/webui/resources/tools/bundle_js_excludes.gni")
+ import("//ui/webui/resources/tools/generate_grd.gni")
+ import("//ui/webui/resources/tools/minify_js.gni")
+ import("//ui/webui/webui_features.gni")
++import("//brave/resources/brave_grit.gni")
```

Comment:
this can be added before `brave_or_default_grit`

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/b8754b2a9bd54be5fe0c2216ea7684b5bd523bad